.circleci/config.yml

```
@@ -97,8 +98,44 @@ jobs:
       - store_artifacts:
           path: results
 
+  coverage:
+    docker:
+      - image: cimg/python:3.12
+    steps:
+      - checkout
+      - run:
+          name: Before install
+          command: |
+            sudo apt-get update
+            sudo apt-get install graphviz graphviz-dev
+      - run:
+          name: Install dependencies
+          command: |
+            python -m venv venv
+            source venv/bin/activate
+            pip install --upgrade pip
+            pip install .[default,extra,test]
+            pip install coverage
+            pip list
+      - run:
+          name: Run tests
+          command: |
+            source venv/bin/activate
+            pytest --cov=networkx --cov-report=html --runslow --doctest-modules --durations=20 --pyargs networkx
```

```suggestion
            pytest --cov=networkx --cov-report=html -n auto --runslow --doctest-modules --durations=20 --pyargs networkx
```

I forgot the xdist flag - let's add that back in too!

---

For some reason, this spawns a bunch of workers that end up failing: https://app.circleci.com/pipelines/github/networkx/networkx/11892/workflows/0cfe8204-036a-42d9-bfb6-75f8c8d4fce0/jobs/20547.